docs/interaction/rpc.md

```diff
@@ -10,118 +10,508 @@ The following methods are available:
+      "id": "CqiBnYCRgkGV2odJPXJTBkoZHXjDNQFJKw7oRYWU1j6g",
+      "outcome": {
+        "gas_burnt": 924119500000,
+        "logs": [],
+        "receipt_ids": [
+          "Cj9eqPrvh9oCAUi6hRxiQjJzsEAa7bQnja3q5U6bX5yC"
+        ],
+        "status": {
+          "SuccessReceiptId": "Cj9eqPrvh9oCAUi6hRxiQjJzsEAa7bQnja3q5U6bX5yC"
+        }
+      },
+      "proof": [
+        {
+          "direction": "Right",
+          "hash": "81vrGGwyr15GvPkhMeV3NcHZLRy6T2z86JQyeyZPe87r"
+        }
+      ]
     }
+  }
 }
 ```
 
-## Transaction Status
+## Validators Endpoint
+
+`validators(blockHash: string)`
 
-`tx(hash: bytes)`: queries status of the transaction by hash, returns FinalTransactionResult that includes status, logs and result: `{"status": "Completed", "logs": [{"hash": "<hash>", "lines": [], "receipts": [], "result": null}]}`
+Queries for active validators on the network and returns details about them and the current state of validation on the blockchain
 
-`http post http://127.0.0.1:3030/ jsonrpc=2.0 method=tx params:=["<base 58 of transaction hash>", "<sender account id>"] id=dontcare`
+- HTTP `POST`
+- URL `https://rpc.nearprotocol.com`
+- method `validators`
+- params
+  - (1) `<base 58 block hash>` (see [status page](https://rpc.nearprotocol.com/status) for a valid `latest_block_hash`)
+
+```bash
+http post https://rpc.nearprotocol.com jsonrpc=2.0 method=validators params:='["8ehA3NYL5uSF8zefbnqnz66twYJ45rfst6SrqBNv7oka"]' id=dontcare
+```
 
+*currently*
+
+```json
+{
+  "id": "dontcare",
+  "jsonrpc": "2.0",
+  "result": {
+    "current_fishermen": [],
+    "current_proposals": [],
+    "current_validators": [
+      {
+        "account_id": "far",
+        "is_slashed": false,
+        "num_missing_blocks": 0,
+        "stake": "200663637577332478904026936259"
+      }
+    ],
+    "next_fishermen": [],
+    "next_validators": [
+      {
+        "account_id": "far",
+        "amount": "200665040940372677611389132209",
+        "public_key": "ed25519:7rNEmDbkn8grQREdTt3PWhR1phNtsqJdgfV26XdR35QL"
+      }
+    ]
+  }
+}
+```
+
+*coming soon ...* (see [`nearcore` PR 1962](https://github.com/nearprotocol/nearcore/pull/1962))
+
+```json
+{
+  "id": "123",
+  "jsonrpc": "2.0",
+  "result": {
+    "current_fishermen": [],
+    "current_proposals": [],
+    "current_validators": [
+      {
+        "account_id": "test.near",
+        "is_slashed": false,
+        "num_expected_blocks": 38,
+        "num_produced_blocks": 38,
+        "public_key": "ed25519:22skMptHjFWNyuEWY22ftn2AbLPSYpmYwGJRGwpNHbTV",
+        "shards": [
+          0
+        ],
+        "stake": "50445149127868051176979985"
+      }
+    ],
+    "next_fishermen": [],
+    "next_validators": [
+      {
+        "account_id": "test.near",
+        "public_key": "ed25519:22skMptHjFWNyuEWY22ftn2AbLPSYpmYwGJRGwpNHbTV",
+        "shards": [
+          0
+        ],
+        "stake": "50445324683980935237733345"
+      }
+    ]
+  }
+}
+```
```

Comment:
you can refer to the full list here - https://github.com/nearprotocol/nearcore/blob/master/chain/jsonrpc/client/src/lib.rs#L185

there are also:
 - chunk(bloch_height / block_hash)
 - gas_price('null' / block_height / block_hash)

Reply:
✅